app/views/contacts/_help_sidebar.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="sidebar" class="<%= sidebar_classes %>">
```

Comment:
You can pull in the partial from the static pages, to avoid duplicating the partial.